qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -238,7 +238,26 @@ def selection(self, html=False):
         return self._widget.selectedText()
 
     def follow_selected(self, *, tab=False):
-        log.stub()
+        def _follow_selected_cb(js_elem):
+            print(js_elem)
+            if js_elem is not None:
+                elem = webengineelem.WebEngineElement(js_elem, tab=self._tab)
+                if tab:
+                    click_type = usertypes.ClickTarget.tab
+                else:
+                    click_type = usertypes.ClickTarget.normal
+
+                # Check to see if we have a real link, to follow before clicking
+                href_tags = ['a', 'area', 'link']
```

Comment:
Why not just an `is_link` element in `AbstractWebElement`? Then that could be used in other places where we check the tag name now too (`AbstractWebElement.click` and `WebKitElement.remove_blank_target` - the latter doesn't check for `link` currently but I think it's okay if it does).